src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js

```diff
@@ -9,8 +9,8 @@ export default BaseAuthenticatorView.extend({
 
     const currentAuthenticator = this.options?.appState?.get('currentAuthenticator');
     const selectedMethod = currentAuthenticator?.methods[0];
-    if (selectedMethod?.type === 'totp') {
-      this.Body = ChallengeOktaVerifyTotpView;
+    if (selectedMethod?.type === 'push') {
```

Comment:
Do you need to update the mock?

Reply:
they are updated ( at least the ones relevant to fastpass UV)

Reply:
Can you show me how it is updated? I can't find it from any mock.

Reply:
https://github.com/okta/okta-signin-widget/pull/1422/files?file-filters%5B%5D=.json